redturtle/bandi/vocabularies.py

```diff
@@ -14,13 +14,7 @@ def __call__(self, context):
         values = api.portal.get_registry_record(
             "tipologie_bando", interface=IBandoSettings, default=[]
         )
-        terms = []
-        for tipologia in values:
-            if tipologia and "|" in tipologia:
-                key, value = tipologia.split("|", 1)
-                terms.append(SimpleTerm(value=key, token=key, title=value))
-            else:
-                logger.error("invalid tipologia bando %s", tipologia)
+        terms = [SimpleTerm(value=x, token=x, title=x) for x in values if x]
```

Comment:
what if editors want to have a term with non ascii chars?

Reply:
is it a problem in python3?

Reply:
it could be.

![image](https://github.com/user-attachments/assets/d56da05b-1ec6-4b76-b08b-250b7f96d509)